pages/faq.md

```diff
@@ -0,0 +1,182 @@
+ 
+<a name="7"><a>
+
+**Under which open source license is AWX available?**
+
+The AWX source code is available under the Apache License 2.0.
+
+<a name="8"><a>
+
+**How can I get involved with the AWX Project?**
+
+The AWX Project is [hosted on GitHub](https://github.com/ansible/awx). We welcome community contributions. Read the [contributor guide](https://github.com/ansible/awx/blob/devel/CONTRIBUTING.md). Join the [Ansible Community Forum](https://forum.ansible.com).
+
+<a name="9"><a>
+
+**Where do I report an AWX bug?**
+
+The AWX project uses GitHub for its issue tracking. You can file your issues [on this page](https://github.com/ansible/awx/issues).
+
+<a name="10"><a>
+
+**How often is AWX released?**
+
+The AWX team currently plans to release new builds approximately every 2 weeks. The AWX team will flag certain builds as “stable” at their discretion. Note that the term “stable” does not imply fitness for production usage or any kind of warranty.
+
+<a name="11"><a>
+
+**What is the governance structure of the AWX project?**
+
+For now, the Red Hat sponsored AWX team will continue to make the majority of decisions for the AWX project, with input from the community. Over time, the team will evaluate its governance structure and choose a structure that balances community needs with product needs.
+
+<a name="12"><a>
+ 
+**If my software runs with AWX, can I say that it is certified to run on AWX or on Red Hat Ansible Automation Platform?**
+
+No. Only Red Hat has the ability to say that software is “certified,” although you may truthfully use the AWX name to describe the relationship between your software and AWX. For more details, consult the [AWX trademark guidelines](https://github.com/ansible/awx-logos/blob/master/TRADEMARKS.md).
+
+<a name="13"><a>
+
+**I want to build my own forked version of AWX. Can I call it AWX? Can I call it Red Hat Ansible Automation Platform?**
+
+No. You may fork AWX like any open source codebase, but you may not use Red Hat trademarks. Red Hat reserves the exclusive right to decide what products can bear those marks. Be sure to read the [AWX trademark guidelines](https://github.com/ansible/awx-logos/blob/master/TRADEMARKS.md) for more information.
+
+<a name="14"><a>
+ 
+**Will contributions to AWX require a “Contributor License Agreement”?**
+
+No. However, all contributions to AWX will require agreement with the Developer Certificate of Origin (DCO) at the time of submission. The text of the DCO can be read in full at [developercertificate.org](http://developercertificate.org/).
+
+<a name="15"><a>
+
+**Does Red Hat recommend AWX for production environments?**
+
+No.
+
+<a name="16"><a>
+
+**Does Red Hat’s Open Source Assurance Program apply to AWX?**
+
+No.
+
+---
+
+# Ansible Community
```

Comment:
I guess this answers my previous question about the FAQ but this change really buries the community part in an old FAQ section. I don't know if I would expect to find this here.